### DISCUSSION TITLE
Set mime-type response on Transmit SSE broadcast messages

### DISCUSSION THREAD

**Discussion by cmjunior** [adonisjs/core#4634] [Help] [answered]
I´m testing the Transmit to emit a SSE, via broadcast, I also implemented a service on angular, to interact with it. It appears to work, but immediatelly after the conection, the browser closes the connection, with the error below:

`EventSource's response has a MIME type ("application/json") that is not "text/event-stream". Aborting the connection.`

Is there any way to set the content type on this broadcast? I could not find where to configure it.

**Comment by RomainLanz** [comment, marked as answer]
Hey @cmjunior! 👋🏻 

The MIME type is already [set to `test/event-stream` by the Transmit package](https://github.com/adonisjs/transmit/blob/develop/src/stream.ts#L70).

If you are receiving something else, it means you have changed it yourself in a middleware or something like that.
The best would be to have a repository with **the minimum amount of code** to reproduce your issue.

**Reply by cmjunior** [reply]
@RomainLanz, you’re absolutely right, thanks in advance. My `forceJsonResponse` was changing the `Accept` header to `application/json`. I had to force the response `Content-Type` to `text/event-stream`.

I noticed that my browser keeps receiving updates, even when I haven't sent any events from the server. I set the `pingInterval` to `false`, but I keep receiving these events. Is this normal?
